### PR TITLE
Nisra content changes

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Various schema content changes outlined in this card [#3205](https://trello.com/c/D0xgMKIt/3205-individual-changes-nisra)

### How to review 
Check content looks ok for each part of the trello card
[Quick launch for NIR schema](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-NIR&url=https://gist.githubusercontent.com/rmccar/c4c691122d1bf135eb5a1dcb8477a225/raw/923fbc23831f78a1b3b8356f7deab5a3edea7dfb/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by rmccar** [comment]
[Individual Changes - NISRA](https://trello.com/c/D0xgMKIt/3205-individual-changes-nisra)

**Review by JamesGardiner — APPROVED** [review]

File: data-source/jsonnet/northern-ireland/blocks/individual/employment/work_location.jsonnet [hunk 1]
```diff
@@ -39,17 +39,17 @@ local question(title) = {
   ],
 };
 
-local nonProxyTitleWork = 'What address do you travel to for your main place of work?';
+local nonProxyTitleWork = 'What is the address of your main place of work?';
 local proxyTitleWork = {
-  text: 'What address does <em>{person_name}</em> travel to for their main place of work?',
+  text: 'What is the address of <em>{person_name}</em>\'s main place of work?',
```
Comment (MebinAbraham) [inline data-source/jsonnet/northern-ireland/blocks/individual/employment/work_location.jsonnet]: We shouldn't be doing possessiveness like this, Welsh doesn't have `'s`. We already have one in `rules.libsonnet` for handling this. `personNamePossessive`

Same applies to all other cases in this PR.

**Review by MebinAbraham — DISMISSED** [review]

File: data-source/jsonnet/northern-ireland/blocks/individual/employment/work_location.jsonnet [hunk 2]
```diff
@@ -39,17 +39,17 @@ local question(title) = {
...
   placeholders: [
     placeholders.personName,
   ],
 };
 
-local pastNonProxyTitleWork = 'What address did you travel to for your main place of work?';
+local pastNonProxyTitleWork = 'What was the address of your main place of work?';
 local pastProxyTitleWork = {
-  text: 'What address did <em>{person_name}</em> travel to for their main place of work?',
+  text: 'What was the address of <em>{person_name_possessive}</em> main place of work?',
```
Comment (MebinAbraham) [inline data-source/jsonnet/northern-ireland/blocks/individual/employment/work_location.jsonnet:52]: The actual placeholder needs to be changed. It is still referencing `personName`

Comment (rmccar) [reply data-source/jsonnet/northern-ireland/blocks/individual/employment/work_location.jsonnet:52]: Oh yeah missed that, fixed now

File: data-source/jsonnet/northern-ireland/blocks/individual/employment/employment_type.jsonnet [hunk 3]
```diff
@@ -58,4 +58,18 @@ local proxyTitle = {
       when: [rules.proxyYes],
     },
   ],
+  routing_rules: [
+    {
+      goto: {
```
Comment (MebinAbraham) [inline data-source/jsonnet/northern-ireland/blocks/individual/employment/employment_type.jsonnet]: When you define at least 1 routing rule, you must define a default rule. Ie currently you have `if`, but need the `else`. The default rule is just a routing rule without a `when` property.

**Review by MebinAbraham — DISMISSED** [review]

**Review by donnap573 — COMMENTED** [review]
Other bug found. New card to be raised

**Review by donnap573 — APPROVED** [review]

**Pull request merged**